echopype/tests/preprocess/test_preprocess.py

```diff
@@ -306,15 +316,16 @@ def test_compute_MVBS():
     # so that when compute_MVBS is performed, the result is a smaller array
     # that increases by a constant for each meter_bin and time_bin
     data = _construct_MVBS_toy_data(
-        nfreq=nfreq,
+        nchan=nchan,
         npings=npings,
         nrange_samples=nrange_samples,
         ping_size=ping_rate * ping_time_bin,
         range_sample_size=range_sample_per_meter * range_meter_bin,
     )
 
     data_log = 10 * np.log10(data)  # Convert to log domain
-    freq_index = np.arange(nfreq)
+    chan_index = np.arange(nchan).astype(str)
+    freq_nom = np.arange(nchan)
```

Comment:
`freq_nom` might be more aptly named `freq_nom_index`, right? That is, unless the intent here was to extract actual frequency_nominal values, not indices